unit-tests/src/org/commcare/CommCareTestApplication.java

```diff
@@ -80,6 +80,10 @@ public PrototypeFactory getPrototypeFactory(Context c) {
         return testPrototypeFactory;
     }
 
+    /**
+     * Get externalizable classes from *.class files in build dirs. Used to
+     * build PrototypeFactory that mirrors a prod environment
```

Comment:
Ah no nevermind I got tripped up by the word "get", but makes sense now. Could make change "Get" to "Pull" for clarification